src/BootstrapDropdownField.php

```diff
@@ -0,0 +1,61 @@
+<?php
+namespace UncleCheese\BootstrapForms;
+
+use SilverStripe\Core\Config\Config;
+use SilverStripe\View\Requirements;
+use SilverStripe\Forms\DropdownField;
+use SilverStripe\i18n\i18n;
+
+/**
+ * Defines a FormField that uses the bootstrap-select JS plugin for making
+ * dropdown fields nice.
+ *
+ */
+class BootstrapDropdownField extends BootstrapFormField {
+
+
+
+	/**
+	 * Builds the form field, sets default attributes, and includes JS
+	 *
+	 * @param array $attributes The attributes to include on the formfield
+	 * @return SSViewer
+	 */
+	public function FieldHolder($attributes = array ()) {
+		if(!Config::inst()->get(BootstrapForm::class, 'bootstrap_select_included')) {
+			$current_locale = i18n::get_locale();
+			Requirements::javascript(BOOTSTRAP_FORMS_DIR."/javascript/bootstrap-select/js/bootstrap-select.min.js");
+			Requirements::javascript(BOOTSTRAP_FORMS_DIR."/javascript/bootstrap-select/js/i18n/defaults-{$current_locale}.js");
+			Requirements::css(BOOTSTRAP_FORMS_DIR."/javascript/bootstrap-select/css/bootstrap-select.min.css");
+		}
```

Comment:
Fix require calls.

Reply:
Fixed in https://github.com/unclecheese/silverstripe-bootstrap-forms/pull/56/commits/034d9e8b441a32fa13ca2271f0ec04a0e6c71525